app/lib/methods/loadMissedMessages.ts

```diff
@@ -1,47 +1,136 @@
 import { ILastMessage } from '../../definitions';
+import { compareServerVersion } from './helpers';
+import updateMessages from './updateMessages';
 import log from './helpers/log';
 import database from '../database';
 import sdk from '../services/sdk';
-import updateMessages from './updateMessages';
+import { store } from '../store/auxStore';
+
+const count = 50;
+
+const syncMessages = async ({ roomId, next, type }: { roomId: string; next: number; type: 'UPDATED' | 'DELETED' }) => {
+	// @ts-ignore // this method dont have type
+	const { result } = await sdk.get('chat.syncMessages', { roomId, next, count, type });
+	return result;
+};
+
+const getSyncMessagesFromCursor = async (
+	roomId: string,
+	lastOpen?: number,
+	updatedNext?: number | null,
+	deletedNext?: number | null
+) => {
+	try {
+		const promises = [];
+
+		if (lastOpen && !updatedNext && !deletedNext) {
+			promises.push(syncMessages({ roomId, next: lastOpen, type: 'UPDATED' }));
+			promises.push(syncMessages({ roomId, next: lastOpen, type: 'DELETED' }));
+		}
+		if (updatedNext && typeof updatedNext !== 'undefined') {
+			promises.push(syncMessages({ roomId, next: updatedNext, type: 'UPDATED' }));
+		}
+		if (deletedNext && typeof deletedNext !== 'undefined') {
+			promises.push(syncMessages({ roomId, next: deletedNext, type: 'DELETED' }));
+		}
+
+		const [updatedMessages, deletedMessages] = await Promise.all(promises);
+		return {
+			deleted: deletedMessages?.deleted ?? [],
+			deletedNext: deletedMessages?.cursor.next,
+			updated: updatedMessages?.updated ?? [],
+			updatedNext: updatedMessages?.cursor.next
+		};
+	} catch (error) {
+		console.error('Error syncing messages:', error);
```

Comment:
log() already does a `console.error`.
It doesn't add `'Error syncing messages:'`, but it's good enough.
Let's remove this one.